rascal-lsp/src/main/rascal/lsp/lang/rascal/lsp/DocumentSymbols.rsc

```diff
@@ -43,7 +43,8 @@ list[DocumentSymbol] documentRascalSymbols(start[Module] \mod) {
     children = [];
 
     top-down-break visit (m) {
-        case decl: (Declaration) `<Tags _> <Visibility _> <Type t> <{Variable ","}+ vars>;`:
+        case amb(set[Tree] _): {/* skip this, and try to match one of the alternatives */;}
```

Comment:
I'm not sure this comment is correct? as it's a `top-down-break` visit so we'll just stop this visit and only look at siblings but not alternatives inside of this amb node?

I mean, I think it's fine to in case of amb node, not to show anything in the outline, but I think the comment is confusing.